_posts/en/newsletters/2024-07-19-newsletter.md

```diff
@@ -0,0 +1,187 @@
+---
+title: 'Bitcoin Optech Newsletter #312'
+permalink: /en/newsletters/2024/07/19/
+name: 2024-07-19-newsletter
+slug: 2024-07-19-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Distributed key generation protocol for FROST:** Tim Ruffing and
+  Jonas Nick [posted][ruffing nick post] to the Bitcoin-Dev mailing list
+  a [BIP draft][chilldkg bip] with a [reference implementation][chilldkg
+  ref] of ChillDKG, a protocol for securely generating keys to be used
+  with [FROST][]-style [scriptless threshold signatures][topic threshold
+  signature] that are compatible with Bitcoin's [schnorr
+  signatures][topic schnorr signatures].
+
+  Scriptless threshold signatures are compatible with the creation of
+  `n` keys, any `t` of which can be used cooperatively to create a valid
+  signature.  For example, a 2-of-3 scheme creates three keys, any two
+  of which can produce a valid signature.  Being _scriptless_, the
+  scheme relies entirely on operations external to consensus and the
+  blockchain, unlike Bitcoin's built-in scripted threshold signature
+  operations (e.g., using `OP_CHECKMULTISIG`).
+
+  Similar to generating a regular Bitcoin private key, each person
+  creating a key for scriptless threshold signatures must generate
+  a large random-like number without disclosing that number to anyone
+  else.  However, each person must also distribute derived shares of
+  that number across the other users so that a threshold number of
+  them can create a signature if that key is unavailable.  Each user
+  must verify that the information they received from every other user
+  was generated correctly.  Several key generation protocols for
+  performing these steps exist, but they assume the key-generating users
+  have access to a communication channel that is encrypted and
+  authenticated between individual pairs of users and that also allows
+  uncensorable authenticated broadcast from each user to all other
+  users.  The _Chill distributed key generation_ (ChillDKG) protocol
+  combines a well-known key generation algorithm for FROST with
+  additional modern cryptographic primitives and simple algorithms to
+  provide the necessary secure, authenticated, and provably uncensored
+  communication.
+
+  Encryption and authentication between participants starts with an
+  [elliptic curve diffie-hellman][ecdh] (ECDH) exchange.  Proof of
```

Comment:
I know how we feel about uppercase names around here, but this still seems a bit egregious. (It's also inconsistent as all previous newsletters had "Diffie–Hellman" in uppercase.)
```suggestion
  [Elliptic-curve Diffie--Hellman][ecdh] (ECDH) exchange.  Proof of
```

Reply:
My decision years ago to go lowercase on eponyms in protocol names was a weird one, for sure.  However, it seems increasingly inconsistent to me to do it only in some cases rather than all cases, so this is an attempt to be more consistent going forward.

How about I leave this as is for this newsletter, but if you feel strongly about it, you open an issue and we can discuss changing the general policy (plus optionally revising already published content).  If you do that, please @ me on the issue so I see it.